core/db/migration/migrate-db.sh

```diff
@@ -16,6 +16,7 @@ echo "Implementing Migrations"
 
 psql -c "select addcol('public','users', 'last_login', 'timestamp', 'current_timestamp');"
 psql -c "select addcol('public','media', 'fine_amount', 'real', '0.50');"
+psql -c "select addcol('public','inventory', 'price', 'money', '1.00' );"
```

Comment:
@stevethomas19 I think we were going to use line 18 to make the prices for the items so we didn't have 3k+ fields to worry about updating. You should be able to use a join with media type and the item in the inventory and have a default price set if it doesn't have a type (might not be necessary if all items have a type)